alerts/ldap_bruteforce.conf

```diff
@@ -0,0 +1,3 @@
+[options]
+threshold_count = 1
```

Comment:
We need to set this to a higher value in prod, but we can use our data to support where exactly we should set this threshold per our specific environment.